packages/teleport-shared/src/utils/uidl-utils.ts

```diff
@@ -424,3 +424,16 @@ export const transformAttributesAssignmentsToJson = (
 
   return newStyleObject
 }
+
+export const traverseUIDLForRootElmNode = (node) => {
+  if (node && node.type) {
+    if (node.type === 'element') {
```

Comment:
this should be replaced with a switch like function:
 - case element -> return node
 - case static/dynamic throw error
 - case conditional/repeat/slot -> return recursive call